components/UserList.js

```diff
@@ -0,0 +1,153 @@
+import { Component } from 'react'
+import fetch from 'isomorphic-unfetch'
+import { withAuth } from 'use-auth0-hooks'
+
+import UserRow from './UserRow'
+
+async function secureFetch (url, accessToken, method = 'get', options = {}) {
+  const res = await fetch(url, {
+    method,
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+      'x-api-key': process.env.API_KEY
+    },
+    ...options
+  })
+  if (res.status >= 400) {
+    const result = await res.json()
+    let message = `Error ${method}-ing user: ${result.message}`
+    if (result.detail) message += `  (${result.detail})`
+    return window.alert(message)
+  }
+  return res.json()
+}
+
+class UserList extends Component {
```

Comment:
Same as https://github.com/ibi-group/otp-middleware/pull/13#discussion_r395105011, it is ambiguous whether the user is a regular OTP user or an admin dashboard user. It feels User can be both here.

Reply:
Yes, we need some further discussions to define this. See https://github.com/ibi-group/otp-middleware/pull/13#discussion_r395087513